pkg/providers/tinkerbell/testdata/cluster_tinkerbell_external_etcd.yaml

```diff
@@ -147,18 +148,18 @@ spec:
       - environment:
           CONTENTS: |
             datasource:
-            Ec2:
-              metadata_urls: ["http://1.2.3.4:50061"]
-              strict_id: false
+              Ec2:
```

Comment:
looks like a space got introduced here. That indentation will make Ec2 a child of datasource

Reply:
Ec2 is a child of datasource so I had to make that change for the test files.